docs/manual/build_system.qbk

```diff
@@ -6,44 +6,105 @@
     file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 ===============================================================================/]
 
-[section:build_system The HPX Build System]
+[section:build_system The __hpx__ Build System]
 
 The buildsystem for __hpx__ is based on __cmake__. CMake is a cross-platform
 build-generator tool. CMake does not build the project, it generates the files
 needed by your build tool (GNU make, Visual Studio, etc) for building __hpx__.
 
-This section gives an introduction on how to use our build system to build HPX
-and how to use HPX in your own projects.
+This section gives an introduction on how to use our build system to build __hpx__
+and how to use __hpx__ in your own projects.
 
 [include build_system/cmake_basics.qbk]
 [include build_system/prerequisites.qbk]
 
-[section:building_hpx Building HPX]
+[section:building_hpx Building __hpx__]
+
+[heading:info Basic Information]
+
+Once __cmake__ has been run, the build process can be started. The __hpx__ build process
+is highly configurable through __cmake__ and various __cmake__ variables influence
+the build process. The build process consists of the following parts:
+
+* [*The __hpx__ core libraries]: This forms the basic set of __hpx__ libraries. The generated targets are:
+  * hpx: The core __hpx__ library (always enabled).
+  * hpx_init: The __hpx__ initialization library that applications need to link
+    against to define the __hpx__ entry points (disabled for static builds).
+  * iostreams_component: The component used for (distributed) IO (always enabled).
+  * component_storage_component: The component needed for migration to persistant storage.
+  * unordered_component: The component needed for a distributed hash table.
```

Comment:
Good point. I will add the missing one. I just listed the ones that are currently created by the "core" target, this probably should be fixed.